WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -107,7 +107,7 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	public void MakeNextCoordinatorScriptDirty()
 	{
 		CoordinatorExtPubKeyCurrentDepth++;
-		if (FilePath is { })
+		if (!string.IsNullOrWhiteSpace(FilePath))
```

Comment:
FilePath == null is used as an indicator, that it is not set. So let FilePath be nullable.

Reply:
Converting it to nullable creates a bunch of more warnings everywhere we use it. Should I just put 
```
		if(FilePath is null)
		{
			throw new ArgumentNullException($"{nameof(FilePath)} is not set.");
		}
```
to every occurence?
Having "" could also be an indicator to set it.

Reply:
I looked into this. Basically, we are hacking around with empty FilePath in tests. As a first step to sorting this out, this PR is good as it is. FilePath should not be null in any case - it is not nullable. This is a safe first step.